### PR TITLE
Add Get-IamPolicy cmdlet.

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/460)

### CODE REVIEW

File: Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿. $PSScriptRoot\..\GcloudCmdlets.ps1
+Install-GcloudCmdlets
+
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GceInstance" {
```
Comment (iantalarico) [inline Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1]: Get-GcIamPolicy?

File: Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    It "should work" {
+        $policies = Get-GcIamPolicy
+        $policies.Count -gt 0 | Should Be $true
+
+        $editors = $policies | Where-Object {$_.Role -eq "roles/editor"}
+        $editors.Members.Count -gt 0 | Should Be $true
+        $editors.Members -contains "serviceAccount:appveyorci-testing@gcloud-powershell-testing.iam.gserviceaccount.com" |
```
Comment (iantalarico) [inline Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1]: What is this? appveyorci-testing@gcloud-powershell-testing.iam.gserviceaccount.com

Can we add this as a const and add a comment?

File: Google.PowerShell/CloudResourceManager/IamPolicyCmdlets.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    /// <para type="link" uri="(https://cloud.google.com/iam/docs/overview)">
+    /// [Google Cloud IAM]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcIamPolicyBinding")]
+    public class GetGcIamPolicyBindingCmdlet : CloudResourceManagerCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to check for Iam Policies. If not set via PowerShell parameter processing, will
```
Comment (iantalarico) [inline Google.PowerShell/CloudResourceManager/IamPolicyCmdlets.cs]: IAM

**Comment by quoctruong** [comment]
Fixed the suggestion. Thanks!

**Pull request merged**